earthquake-engineering-and-structural-dynamics.csl

```diff
@@ -91,7 +91,7 @@
     </layout>
   </citation>
   <bibliography et-al-min="7" et-al-use-first="6" second-field-align="flush" entry-spacing="0">
-    <layout suffix=".">
+    <layout suffix=". ">
```

Comment:
could you remove the space here? It's probably not going to make a difference in many circumstances, but it doesn't make any sense to put a space after the final period.